### ISSUE TITLE
evdi_encoder.c:57:11: error: format not a string literal and no format arguments

### ISSUE THREAD

**Issue by thx1111** [DisplayLink/evdi#277]
Arch
linux 5.11.13.arch1-1
```
Making package: evdi-git 1.9.1.r4.gb0b3d13-1 (Sun 11 Apr 2021 12:32:16 PM MDT)
...
/var/build/evdi-git/src/evdi/module/evdi_encoder.c: In function ‘evdi_encoder_init’:
/var/build/evdi-git/src/evdi/module/evdi_encoder.c:57:11: error: format not a string literal and no format arguments [-Werror=format-security]
   57 |           DRM_MODE_ENCODER_TMDS, dev_name(dev->dev));
      |           ^~~~~~~~~~~~~~~~~~~~~
cc1: some warnings being treated as errors
...
```

**Comment by thx1111** [comment]
On Arch Linux, with pacman 5.2.2-3, currently in testing, /etc/makepkg.conf now sets `-Werror=format-security` by default in `CFLAGS=`.

**Comment by ffgiff** [comment]
Also an issue on Fedora 36

**Comment by synaptics-lspintzyk** [comment]
Seems to be fixed in evdi-1.14.8. Closing.
